docs/source/AdministratorGuide/Tutorials/diracSE.rst

```diff
@@ -0,0 +1,180 @@
+.. _tuto_install_dirac_se:
+
+===============================
+Install a DIRAC Storage Element
+===============================
+
+Pre-requisite
+=============
+
+You should have a machine setup as described in :ref:`tuto_basic_setup`, and be able to install dirac components. For simple interaction with the StorageElement using ``dirac-dms-*`` commands, you should also have a working FileCatalog.
+
+
+Tutorial goal
+=============
+
+The aim of the tutorial is to do a step by step guide to install a DIRAC StorageElement. By the end of the tutorial, you will be able to have a fully functional storage element that can be addressed like any other storage.
```

Comment:
I would add 2 links:
- https://dirac.readthedocs.io/en/latest/AdministratorGuide/Resources/Storage/index.html#storageelement
- https://dirac.readthedocs.io/en/latest/AdministratorGuide/Configuration/ConfReference/Systems/DataManagement/Services/StorageElement/index.html

Reply:
Good idea will do

Reply:
However we had agreed to remove the ConfReference part, and replace it with the doc generated from the confTemplate, so I won't add that link but can document the ConfTemplate for the SE

Reply:
Added that